docs/deploying_apps/deploying_rails.md

```diff
@@ -60,6 +60,17 @@ You can add another instance of your app by running:
 
 ``cf scale APPNAME -i 2``
 
+## Web servers
+
+By default, the Cloud Foundry buildpack for Rails 4 [runs `bin/rails server`][cf-buildpack-rails] to
+spin up the application, which will use WEBrick as a server.
+
+You may want to use a different server in production. The Cloud Foundry docs have
+[more information on configuring a production server][cf-docs-prodserver]
+
+[cf-buildpack-rails]: https://github.com/cloudfoundry/ruby-buildpack/blob/1f0ac3ce10866390d161c3f27e71d64890859454/lib/language_pack/rails4.rb#L27
```

Comment:
will this link still be valid when we update the buildpack? would it need maintenance? If so, could it be replaced with something that's more useful/relevant for the future?